lib/hooks/userspace/hook-grpc.js

```diff
@@ -25,6 +25,15 @@ var agent;
 
 var SUPPORTED_VERSIONS = '0.13 - 0.15';
 
+function findFirstIndex(arr, predicate) {
```

Comment:
Let's bring in [lodash.find](https://lodash.com/docs#find) for this. If only [Array.prototype.find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) was present in `0.12` 😿